### PR TITLE
Fix failing mesheryctl unit tests

### PR DESCRIPTION
**Pull request by nishantwrp**
Signed-off-by: Nishant Mittal <mittalnishant14@outlook.com>

**Description**

This PR fixes n/a. Fix failing mesheryctl unit tests along with some minor improvements int tests.

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by nishantwrp** [comment]
Still need to fix one failing test.

File: models/meshery_application.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ type MesheryApplication struct {
 	UserID        *string        `json:"user_id" gorm:"-"`
 	Location      isql.Map       `json:"location"`
 	Type          sql.NullString `json:"type"`
-	SourceContent []byte
-	UpdatedAt     *time.Time `json:"updated_at,omitempty"`
-	CreatedAt     *time.Time `json:"created_at,omitempty"`
+	SourceContent []byte         `json:"source_content"`
+	UpdatedAt     *time.Time     `json:"updated_at,omitempty"`
```
Comment (leecalcote) [inline models/meshery_application.go:52]: NIce.

**Review by leecalcote — APPROVED** [review]
LGTM

**Comment by leecalcote** [comment]
@nishantwrp, my hero. 🦸

**Pull request merged**